### ISSUE TITLE
XScreensaver TesselImage does not display properly

### ISSUE THREAD

**Issue by uli42** [ArcticaProject/nx-libs#879]
Start an nx session, run `xscreen-saver-demo`, select the `TesselImage` screensaver. The watch the preview window or press Preview to see it fullscreen.

You will notice unfilled or wrongly filled regions like in the screenshot. Compare this to a the same test on a real X server where you only see regions that are filled properly. Sometimes you have to to retry several times before the artefacts appear.

![grafik](https://user-images.githubusercontent.com/6320356/70089718-08737800-1619-11ea-8c52-831f9bd4a0c7.png)

**Comment by georgydunaev** [comment]
Moreover, it consumes a lot of power, 
Fans on my new&fast ThinkPad rotates as crazy when this particular screensaver is turned on
It is dangerous because laptop become hot!

**Comment by uli42** [comment]
Yes, but that's not something we can solve in NX. It also consumes lots of
power without NX...
